chain/store/store.go

```diff
@@ -971,7 +969,24 @@ func (cs *ChainStore) AddToTipSetTracker(ctx context.Context, b *types.BlockHead
 	return nil
 }
 
-func (cs *ChainStore) PersistBlockHeaders(ctx context.Context, b ...*types.BlockHeader) error {
+func (cs *ChainStore) PersistTipset(ctx context.Context, ts *types.TipSet) error {
```

Comment:
Not something we want to refactor now, but `PutTipSet` and `PersistTipSet` are confusing.